lib/Officer.php

```diff
@@ -37,6 +37,19 @@ public function getAllOfficersPatrollingAllLots(){
         return $results;
     }
 
+     // COUNT ALL OFFICERS
+     public function getCountOfOfficers()
+     {
+         $this->db->query("
+             SELECT COUNT(Officer_ID) AS NumberOfOfficers
+             FROM Officer;
```

Comment:
Changed this to `Officer` from `officer`. My mac is case sensitive. :/

Reply:
Ah yeah it's working fine on mine just let me know which ones I'll need to fix